src/hashcat.py

```diff
@@ -0,0 +1,47 @@
+import subprocess as sp
+def hashcat(hash, ids, wordlist):
+    for possible_type in range(len(ids)):
+        command = f"hashcat -a 0 -m {ids[possible_type]} {hash} {wordlist}" # >> Can change for better optimization etc.....
+        # Hashcat64.exe for windows
+        try:
+            sp.check_call(
+                command,
+                shell="True",
+                # cwd='/mnt/p/Hashcat' << For windows, you must put the hashcat directory for it to work. Also the wordlist must be in the directory or a full path.
+                # stderr=sp.STDOUT
+            )
+        except sp.CalledProcessError as error:
+            if "'hashcat' is not recognized as an internal or external command" in str(error):
+                print("Hashcat not in PATH / dir") ; return False
+            if "./hashcat.hctune: No such file or directory" in str(error):
+                print("Invalid wordlist") ; return False
+            print("Couldn't crack")
+            continue
+
+        try:
+            possible_output = str(
+                sp.check_output(
+                    f"{command} --show", shell=True, cwd='/mnt/p/Hashcat'
+                ).strip()
+            ).replace("'","")
+        except:
+            print("Hash not cracked :/")
+        else:
+            if not "No hashes loaded." in possible_output:
+                print(f"\nHashcat Success ¦ Type {ids[possible_type]} ¦ {possible_output.split(':')[1]}")
+                return True
+
+    return False

```

Comment:
At this point, our program should hand over all control to HashCat & kill itself, we don't need to exist if we're handing it off to hashcat so returning False shouldn't be happening here :))